### DISCUSSION TITLE
FYI New ORP Probe from Atlas Scientific

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#951] [General]
I've been having an ongoing conversation with Jordan, a tech support person at A-S regarding instability of the readings from my ORP probe.  The probe is a Gen 3 model with an SMA connector that was installed by A-S.  It's been in service for about 3 months and never seemed provide reliable readings for more than a few days at a time.

I've done a fair amount of troubleshooting and modification to the installation to reduce/eliminate static, interference from other electrical lines, etc. to no avail.  Jordan finally asked me if the pool is saltwater.  Yep, it is.  To which Jordan replied as follows (and this is a direct quote).

_Oh..._
_That explains what's going on._
 
_A few weeks ago, I had a long conversation with a pool company, and they were talking about the need for gold ORP probes._
_For reasons that are unclear to me, platinum ORP probes behave differently in saltwater pools._
_In a pool that uses old-school chlorine tablets, the ORP readings go to ~800+_ 
_But in a saltwater pool, platinum ORP readings are ~40% lower._
 
_The timing of all this is quite unusual._
_Because, behind the scenes, I had been working on a new type of ORP probe for the oil industry._
_A gold ORP probe._
_(Because a platinum ORP probe can react with the oil like a catalytic converter, VERY BAD!)_

Jordan suggested I try their brand spanking new Gen 3 ORP probe that uses gold instead of platinum.  He offered me a discount on the probe since I've been having problems with my probe & been working with him for the past few months.  This morning I ordered the new probe & hope to have it installed in a week or so (the usual delivery time from A-S).

I plan to report my experience with the gold probe once it's been installed & running for a week or so.  

In the interim, if you are getting an ORP probe from A-S and have a saltwater pool, you may want to consider getting the gold probe instead of the platinum model.  A link to the probe is below.  Like the other Gen 3 probes, you can get it with the SMA connector already installed, potentially saving some time & headaches.

https://atlas-scientific.com/probes/gen-3-industrial-orp-probe-gold/

**Comment by johnny2678** [comment]
wow - there are so many memes you can use for this! 

https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaXQ0MTB2NDA2NmxzNm5lY3RoOG8wdHdzZDRtOGJjMzRmcHliMWJmdiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/4SQMqhWzUA0Fi/giphy.gif

This is great info @FlaMike !!

My probes were working fine but I decided to change them annually back in march since I was just running the cheap consumer probes.  This year, for some reason, it did not like the new probe.  I followed your issues and went out and bought some rubber washers to try and isolate any potential interference and in the process, disconnected and reconnected the ORP probe may times.

At least for now, it seems to be stable as of May 1.  But you can see the ~2 months of instability below starting in March when I changed the probes.

![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/0f9aa741-f2fb-4245-ad1a-e17901cb9376)

Looks like the new gold probes requires a 3/4 NPT tap out - which I've avoided all this time 😂, but if it can provide stability and many years of use, I'd have to take another look at doing that.

**Reply by FlaMike** [reply]
@johnny2678, I hope your probe has truly stabilized & keeps you happy throughout the swim season & beyond!  Until I received the note from Jordan at A-S last week, I had no idea their ORP probes could have problems with saltwater pools.  I'm hoping the gold probe is the answer to my ORP dreams.

Great meme, BTW.  You put a real smile on my face!!  In fact, I laughed out loud :-)

Based on Jordan's email to me, I don't know if they're planning to offer gold probes in a smaller form factor that would fit your flow cell without modification.  I held my breath & did the drill & tap shuffle.  @rstrouse gave me some great pointers that made the exercise less stressful and more successful.  He said not to even turn on the drill press & just turn the chuck by hand while applying gentle pressure on the drill bit.  You'll know if you're applying too much downward pressure because you won't be able to turn the chuck.  It look a while & was a bit tiring, but nothing cracked on the flow cell.  Tapping is much more straightforward.  Obviously, if you don't need to do the drill & tap, life is somewhat easier.

Jordan said he tested the probe I purchased this morning & it's working like a champ.  I hope to report the same after I get some experience with it.

**Reply by FlaMike** [reply]
Initial  update...

Late last week installed the gold ORP probe.  After it "settled in" for a few hours (gradually & smoothly increasing its reported ORP value), the probe appears to be working in my pool, at least. better than the platinum ORP probe ever did.  Reported levels do not fluctuate wildly as the platinum probe did.

It's  still very early in my experiment, but I'm hopeful this probe will do the trick for me.  Of course,  we had a ton of  rain over the weekend & the pool's  base chemistry was a bit off as a result.  Still working on getting everything to a happy & quiescent state, but the ORP readings make me think the journey may be back on track.

@johnny2678 I want to let you know that I've continued my conversations with Jordan at A-S & he indicated they may be working on the lab-grade gold ORP probe.  Perhaps you won't have to drill your flow cell after all!

I'll provide a more detailed experience report once I've had a few weeks of experience with the gold probe.  At this time, I am cautiously optimistic.

**Reply by johnny2678** [reply]
great to hear - I'm making due with my old consumer probe but will definitely be on the look out for a lab grade gold probe when it's time to rebuy next year.  Please let them know of my interest!

**Reply by FlaMike** [reply]
Will do!

**Reply by FlaMike** [reply]
Update du jour on gold ORP probes.  I've just been informed that Atlas-Scientific expects to have the lab-grade probes available for purchase in July.  If you anticipate needing a lab-grade ORP probe in the near future, you may want to keep an eye on their web site.  

Nothing much else to report on my experiment since yesterday.  In my pool the gold probe readings appear to be stable and more-or-less as one would expect.

**Reply by johnny2678** [reply]
Great timing! Thanks for the update.

**Reply by FlaMike** [reply]
OK, I've been running the industrial gold ORP probe for almost two weeks.  In my pool (saltwater), the gold probe is much more stable & its output appears to be much more reliable than the platinum probe ever was.  Had the gold probe been available and recommended for saltwater pools several months ago and I purchased it originally, I likely would have saved myself much grief & hand-wringing.  In addition, I may have saved the fine folks who crafted, update, & support njspc & REM many posts as I was seeking help with my chemistry challenges.

In short, if you  have a saltwater pool & are looking for an ORP probe, it may very well be worthwhile to pay the premium for the gold ORP probe rather than the platinum one.

I do not know of is any advantage to using the gold probe in non-saltwater pools, but the folks at A-S do not indicate it is needed. You may want to check with them prior to purchasing, however.

**Comment by TopJim** [comment]
I was eager for Atlas’ lab grade gold ORP probe, but no sign of it exists, yet. I bought Sensorex’s gold lab grade probe for $150, shipped. Will report back on my results.

**Reply by FlaMike** [reply]
Gee, I have no idea what happened to A-S' lab grade gold probe.  The tech support guy who was helping me seemed confident 6-7 weeks ago that the lab probe would be available in July.  I've not looked or inquired since my last posting about it.  Perhaps a note to tech support if you're in the market would be worthwhile?

I hope the Sensorex probe works as well for you as well as A-S' gold industrial probe works for me--it's been rock solid since I installed it.

**Reply by TopJim** [reply]
Atlas now has a lab grade gold ORP sensor for sale. $99. My Sensorex gold one has been fairly stable so far. No "crashing" or inverse-sensing like the platinum one did. I'll get this Atlas one and see how it does. Still collecting data to see if there's some correlation between FC and mV, but seems useful so far, unlike the platinum.

**Reply by FlaMike** [reply]
Glad to hear your Sensorex gold probe is working well and that A-S finally has their lab-grade gold probe available.  Good to have solid options!

I'll be interested to hear about your A-S lab grade experience, whenever you get one.  Not sure why it took so long to get gold probes for saltwater pools, but based on a very limited sample of folks reporting, they seem to be far superior to platinum.

**Reply by TopJim** [reply]
I like the Atlas probe better than the Sensorex. It seemed like the sensorex needed "seasoning" whereas the Atlas probe worked correctly right out of the box. Sensorex started at 400 and then over 3 weeks gradually rose to the "correct" reading of 750. When I put the Atlas probe in, it immediately read 750. I have the Sensorex in my spa and the Atlas in my pool, separate bodies with roughly the same chemistry. I run pH of 7.6 and FC of 4 with CYA brought to 25 at the start of each season, so really, CYA could be anywhere from 25 to 0. So I target an ORP of 750 which corresponds to manual testing of 4 FC. Anyway long story short: when my Sensorex needs replacing I will be getting the Atlas probe.

**Reply by FlaMike** [reply]
Thanks for posting your experience with the two probe options!  It's a little surprising the two probes behave differently initially, but eventually agree.  In any event, very helpful information all around!  I guess the bottom line is to stick with A-S & use gold if you have a saltwater pool.

**Reply by FlaMike** [reply]
Question for all, especially @TopJim @johnny2678 and, of course, @tagyoureit and @rstrouse 

As you may or may not have seen in my post a couple of days ago, when I  was re-installing a newly-calibrated Industrial pH probe, I cracked my flow cell.  Major bummer!  I've ordered what I'm hoping is  a direct replacement for the IPS--it's branded as Hayward & much cheaper if all you need is the cell itself.  It should be here soon, so I'll update that thread soon.

The current cell cracked at the threads.  Because I was installing a Industrial probes, I drilled & tapped the stock IPS cell.  While it worked fine for a year, I assume the thread area cracked because the walls were thinner than normal due to the drilling & tapping.

Not wanting to repeat the cell replacement process, I'm thinking seriously of switching to Lab grade pH & gold ORP probes.  Obivously, they still need to be installed in the flow cell.  Of  course, the Industrial probes are recommended by the creators of njsPC, so there isn't anything in the Wikis about using the lab grade alternatives.

If I understand A-S' offerings correctly, I will need to use their Probe Pipe Fittings with the 1/2" NPT fitting to attach them to the flow cell https://atlas-scientific.com/connectors/probe-pipe-fitting/   I guess I need to get their M-to-F SMA Extension cables  https://atlas-scientific.com/cables/sma-mf-extension-cable/ and finally some Coax Seal at the extensions' junctions for waterproofing https://atlas-scientific.com/cables/coax-seal/

Is this what you use for the Lab grade A-S probes?  Do these probes work well for you & last 2-3 years?  If so, I'm in!  I just don't want to go through another Ooops that causes me to start over with yet another flow cell.

Thanks for you help & feedback!!!

**Reply by johnny2678** [reply]
It won't be an oops.  I've been running consumer/lab grade since I put my flow cell in back in 2019.  Basically because I was too scared I would crack the cell by drilling it out for the industrial probes 🤣

You might need to change them out every 2 years instead of every 4-5, but the cost makes up for that.  Super easy to swap them out and recalibrate with the pipe fittings.

The parts you identified are exactly what I use in my setup.

**Reply by FlaMike** [reply]
Great to know & thanks for confirming the parts list!

**Comment by TopJim** [comment]
Working fine for me but it’s been about 6 mos with the Atlas gold ORP, about a year for the pH. Also my flow cell is in my basement, so protected from the environment. Don’t know if “lab grade” is suitable in the outdoors. Davey JonesOn Apr 15, 2025, at 16:05, FlaMike ***@***.***> wrote:﻿
Question for all, especially @TopJim @johnny2678 and, of course, @tagyoureit and @rstrouse
As you may or may not have seen in my post a couple of days ago, when I  was re-installing a newly-calibrated Industrial pH probe, I cracked my flow cell.  Major bummer!  I've ordered what I'm hoping is  a direct replacement for the IPS--it's branded as Hayward & much cheaper if all you need is the cell itself.  It should be here soon, so I'll update that thread soon.
The current cell cracked at the threads.  Because I was installing a Industrial probes, I drilled & tapped the stock IPS cell.  While it worked fine for a year, I assume the thread area cracked because the walls were thinner than normal due to the drilling & tapping.
Not wanting to repeat the cell replacement process, I'm thinking seriously of switching to Lab grade pH & gold ORP probes.  Obivously, they still need to be installed in the flow cell.  Of  course, the Industrial probes are recommended by the creators of njsPC, so there isn't anything in the Wikis about using the lab grade alternatives.
If I understand A-S' offerings correctly, I will need to use their Probe Pipe Fittings with the 1/2" NPT fitting to attach them to the flow cell https://atlas-scientific.com/connectors/probe-pipe-fitting/   I guess I need to get their M-to-F SMA Extension cables  https://atlas-scientific.com/cables/sma-mf-extension-cable/ and finally some Coax Seal at the extensions' junctions for waterproofing https://atlas-scientific.com/cables/coax-seal/
Is this what you use for the Lab grade A-S probes?  Do these probes work well for you & last 2-3 years?  If so, I'm in!  I just don't want to go through another Ooops that causes me to start over with yet another flow cell.
Thanks for you help & feedback!!!

**Reply by johnny2678** [reply]
I don't know if they are rated for outdoor use either, but I have mine in a south facing project box above my pool equipment, and they seem to do ok in the FL sun. The pool water in the flow cell keeps them cool 😂

**Reply by FlaMike** [reply]
Thanks for your feedback as well.  According to A-S' web page description, the lab grade probes can be immersed up to the SMA connectors indefinitely.  I'll need the extension cables, hence the coax seal in the parts list.  So, I'm not concerned about water intrusion-related issues.

I'm in northwest Florida.  The flow cell is on the east wall of the house, so it's  spared the hottest sun.  And I've generally covered the probes with foam pipe insulation to help prevent freezing in the winter, and possibly keep them cooler in the summer.  We actually had 5" of snow on the beach this year :-(  Took nearly a week to melt, too.

I don't shut the pool down so the freeze protect keeps the water moving, including the flow cell reservoir, if the air temperature reaches around 37 F.  Never frozen a probe yet (not even the Hayward mess at another house).  Hopefully, the trend is my friend.

Like you, I decided the cost is more-or-less a wash given the lab probe's lower cost, but shorter life.  The thought of going through the process of drilling & tapping another flow cell (and then possibly cracking it at the threads months later) is making me wary of another go-around.  I don't have a drill press & had to inconvenience a friend to use his.  I don't want to bug him again (and again?).  I suspect the project leads have much more experience in the NPT world and may rightly disagree.  Like many elements of the pool controller, it was new to me.

Thanks for sharing!  The adventure continues...

**Comment by TopJim** [comment]
Part of the problem (IMO) is that Atlas' Industrial probes are not NPT. They are BSPT. BSPT uses an O-ring for sealing, whereas NPT uses a taper for sealing. Also the threads are different. So you have a BSPT thread (the probe) going into a NPT hole (the flow cell). To do this correctly, you'd need a BSPT tap and and appropriately-sized drill. What I did with mine (when I was using Industrial) was get a NPT to BSPT adapter and O-ring, ala:
https://www.troublefreepool.com/threads/threads-on-atlas-orp-industrial-sensor.290534/

**Reply by FlaMike** [reply]
Wow!  I had the exact same experience with both of my industrial probes from A-S.  Threads were a mess, leaking, & tore up the teflon tape whether I used a little or a lot.  I cleaned up the threads with a tap, to no avail.  I decided a few drops of leakage weren't worth the headache & looked the other way.  I'm guessing I over-tightened the pH probe over the weekend because water was weeping out as I screwed in the probe--the flow was off at the time.   Anyway, I'm hoping the adapter for the lab probe works.  It sounds like that's what @johnny2678 is using.  Oh, what a world!!!  As in the TFP thread you linked, ORP was a mess until I got the gold probe.  Since then it's been good.  Fingers crossed going forward.

**Reply by FlaMike** [reply]
@johnny2678 Did you do anything other than add teflon tape to the "NPT" part of the Atlas pipe fitting, screw it into the flow cell, & install the probes into the fitting?  Did you have any leak issues?  I'm trying to set my expectations appropriately for the replacement flow cell, probes, etc.  I'm a bit concerned about the NPT vs. BSPT issue for the pipe fitting & am hoping it's a non-issue.

Thanks!

**Reply by johnny2678** [reply]
@FlaMike yep, just the stock adapters and O ring that comes with it in my setup and no leaks for 6 years.

![image](https://github.com/user-attachments/assets/4419a31b-e996-400d-a213-e87a68bb0f88)

**Reply by FlaMike** [reply]
Now *that's* a happy news story!  Hoping to duplicate your success!  Everything looks clean as a whistle, too!!  Well done!

**Reply by johnny2678** [reply]
That's because I zoomed in so you couldn't see the mess of parts I left in the bottom of the box and the paper towel I used to plug and unused hole to keep bugs out 😂![image](https://github.com/user-attachments/assets/26846ff3-a32e-4d5f-bb3f-1d0843c16977)

**Reply by FlaMike** [reply]
Framing is everything, but it still looks pretty sweet to me.  My soon-to-be ex flow cell is outside & screwed to the wall.  I don't think it looked as good as yours on day 1.

BTW, I looked at A-S' specs on the lab pH & ORP probes.  Their acceptable temperature range is -5 to 100 C for one of them & 1 to 99 C for the other.  I honestly don't recall which is which, but the water never approaches those extremes (ice pool and steam supplier?).  Even in the worse cold here, I doubt the water has even approached 40 F or above 94 F.  Much smaller range than the probe can handle.  Now the *air* temperature here has been in the 20s F, so below either probe's limits.  Hope the air doesn't freeze the little buggers.  I could always remove them in an extended cold snap.  Pretty sure the air hasn't approached 212 F here in an eon or two as well.

**Comment by TopJim** [comment]
Probe—>#14 o-ring—>adaptor—>teflon tape—>flow cellNo leaks. Note that the adapter came with a gasket, but I found it to be unsuitable, so got the O-ring from Ace. Davey JonesOn Apr 16, 2025, at 12:33, FlaMike ***@***.***> wrote:﻿
@johnny2678 Did you do anything other than add teflon tape to the "NPT" part of the Atlas pipe fitting, screw it into the flow cell, & install the probes into the fitting?  Did you have any leak issues?  I'm trying to set my expectations appropriately for the replacement flow cell, probes, etc.  I'm a bit concerned about the NPT vs. BSPT issue for the pipe fitting & am hoping it's a non-issue.
Thanks!

**Reply by FlaMike** [reply]
Thank you.  Sad that you have to get a different o ring.  I consider myself braced appropriately.

**Reply by TopJim** [reply]
Hope I’m not misleading you, but all the talk about BSPT is for the industrial probe only. Lab grade probes work fine right out of the box.

**Reply by FlaMike** [reply]
Thanks for the clarification!   I received the Hayward CAX-20272 flow cell (body only) from Amazon & it appears  to be the same as the IPS cell body with the same input & output holes pre-drilled and tapped as the IPS unit.

I still have the input "adapters" for the Lab grade probes, complete with o-rings that came with the IPS complete kit.  The appear to fit 1/2" probes and seem to match the threads on the cell itself.  The flow switch and exit port valves also seem to work with the originals from IPS--I did a quick dry-fit check without any surprises.

I am going  to try the Lab grade probes this time around & hope they last 2 years +/-.  If they do, it will save the headache of drilling & tapping for the Industrial probes & hopefully, I won't ever crack a flow cell again!

Thanks for all of your feedback & information!  I'm about to place an order for round 2 of my flow cell with A-S & hope to be back in business sometime next week.

**Reply by FlaMike** [reply]
@johnny2678 Do you know what size valves you installed?  are they 1/4" NPT to 3/8" John Guest?  Do you recall where you purchased them?

Thanks!

**Reply by johnny2678** [reply]
My flow cell came with them preinstalled. 1/4 by 3/8 valve sounds right. Certainly the 3/8 side.

**Comment by FlaMike** [comment]
Thank you for your reply.  I suspected it was 1/4".  They're very
challenging to find--with 3/8" John Gest.  I just ordered a 1/4" to 3/8"
NPT to John Gest adapter and a 3/8" to 3/8" JG inline valve.  Hopefully, it
will work out ok. 🤞

Thanks again for your help!

**Comment by FlaMike** [comment]
@johnny2678   Quick question for you...  You're running the Gold ORP Probe, aren't you?  The reason I am asking is I ordered the lab grade gold probe, but the part number on the box is for the platinum probe.  However, the probe  has the same blue barrel at the top of the probe and the same tip as yours does.  The platimum probe, as pictured on the A-S web site is slightly different as  pictured here.  A-S hasn't responded to my query & I'm holding off on installtion until I can confirm what probe they actually sent to me.  Any insight you can share is appreciated greatly.

Thanks!!!!

Lab Grade Platinum Probe:
![image](https://github.com/user-attachments/assets/6b8563fc-65a0-4cd7-83c5-a5ff8d19d713)

Lab Grade Gold Probe:
![image](https://github.com/user-attachments/assets/69c11d7c-962f-4fe5-b11d-9ebe59dd2b8e)

**Reply by FlaMike** [reply]
Just to follow-up...  A-S eventually got back to me & confirmed the probe is the gold one.  Whew!  I just finished calibrating & installing everything & am once again in "business."  Whew!  I'm really looking forward to starting swim season (tomorrow, I hope!) & not festering about pool chemistry.

Looking back, I was  hesitant to use the industrial probes, only because of my concerns about the drill & tap shuffle. The cell & I survived the drill & tap, but the process of removing & re-installing the industrial probes eventually did me in--I cracked the flow cell at the threads when re-installing the pH probe.  Now that I have installed  the lab grade probes I am much happier.  Even if they only last two years, I don't believe I am risking the flow cell when I recallibrate or replace them.  Since the valves & probe connections are "permanently" installed in the flow cell, it's  not at risk (I hope!).  Furthermore, I've not removed any acrylic from the cell, so the threaded areas are thicker than after the drill & tap shuffle.

I'm hoping the trade-off is even over the long haul.  Of course, your mileage & preferences may vary.

Happy swimming, y'all!

BTW, if anyone reading this is interested in lightly used (1 year or less) industrial pH, platinum and gold ORP probes, I've got a great deal for you--if you pay shipping +/-, they're yours.  Just send me a note.  Crazy Mike's prices are *insane!*

**Reply by johnny2678** [reply]
@FlaMike sorry for the late response.  I am running the gold probe. I still have my platinum probe and can take a picture side-by-side if you like. Sounds like A-S got you sorted, but let me know. Cheers!

**Comment by FlaMike** [comment]
No worries at all.  As you saw, all good here at the moment & everything is
up & running smoothly again.  Your feedback & support have been helpful &
appreciated greatly throughout my journey.

Enjoy your build & its assistance in keeping your pool clean & safe.  Have
a super day & weekend!
